package.json

```diff
@@ -75,9 +75,10 @@
     "postbuild": "npm-run-all --parallel copy:*",
     "postinstall": "node postinstall.js",
     "start": "node build/server.js",
-    "test": "npm run test:server",
+    "test": "npm run test:server; npm run test:client",
```

Comment:
I think you can delete this command. It looks like an old command that does no longer do what you will expect to do (testing the built JS files).